homeassistant/components/prusalink/coordinator.py

```diff
@@ -9,6 +9,7 @@
 from time import monotonic
 from typing import TypeVar
 
+from httpx import ConnectError
```

Comment:
I was wondering why not use pythons error class but the pyprusalink library uses httpx, so this makes sense